renku/service/config.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Renku service config."""
+import os
+import tempfile
+from pathlib import Path
+
+INVALID_PARAMS_ERROR_CODE = -32602
+INTERNAL_FAILURE_ERROR_CODE = -32603
+
+API_VERSION = 'v1'
+
+SWAGGER_URL = '/api/docs'
+API_URL = os.getenv(
+    'RENKU_SVC_SWAGGER_URL', '/api/{0}/spec'.format(API_VERSION)
+)
+
+UPLOAD_FOLDER = tempfile.TemporaryDirectory()
+
+CACHE_UPLOADS_PATH = Path(UPLOAD_FOLDER.name) / Path('uploads')
+CACHE_PROJECTS_PATH = Path(UPLOAD_FOLDER.name) / Path('projects')
+
+ALLOWED_EXTENSIONS = {'txt', 'pdf', 'csv'}
```

Comment:
? why would we want to impose this?

Reply:
Because we want security?

Reply:
it's not very easy to know ahead of time all the possible extensions people will want to upload. Besides, I don't think enforcing something as easily changed as a file extension is a particularly robust security practice.